client/state/action-types.js

```diff
@@ -127,6 +127,10 @@ export const SITE_MEDIA_STORAGE_RECEIVE = 'SITE_MEDIA_STORAGE_RECEIVE';
 export const SITE_MEDIA_STORAGE_REQUEST = 'SITE_MEDIA_STORAGE_REQUEST';
 export const SITE_MEDIA_STORAGE_REQUEST_SUCCESS = 'SITE_MEDIA_STORAGE_REQUEST_SUCCESS';
 export const SITE_MEDIA_STORAGE_REQUEST_FAILURE = 'SITE_MEDIA_STORAGE_REQUEST_FAILURE';
+export const SITE_DOMAINS_RECEIVE = 'SITE_DOMAINS_RECEIVE';
+export const SITE_DOMAINS_REQUEST = 'SITE_DOMAINS_REQUEST';
+export const SITE_DOMAINS_REQUEST_SUCCESS = 'SITE_DOMAINS_REQUEST_SUCCESS';
+export const SITE_DOMAINS_REQUEST_FAILURE = 'SITE_DOMAINS_REQUEST_FAILURE';
```

Comment:
I missed this on my last PR too :) We alphabetize the actions, so failure should be before success.